tools/webhook_test.py

```diff
@@ -43,7 +38,7 @@ def set_init(webhook_type):
                 "player_level": 30,
                 "latitude": 33.980823,
                 "longitude": -81.052988,
```

Comment:
Consider using the defaults from the PA wiki: 37.7876146,-122.390624 for default location values.  :)